Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet.iOS/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Reflection;
+using WebKit;
+using Xamarin.Forms;
+
+// Information about this assembly is defined by the following attributes.
+// Change them to the values specific to your project.
+
+[assembly: AssemblyTitle("Contoso.Forms.Puppet.iOS")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("")]
+[assembly: AssemblyCopyright("")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+
+// The assembly version has the format "{Major}.{Minor}.{Build}.{Revision}".
+// The form "{Major}.{Minor}.*" will automatically update the build and revision,
+// and "{Major}.{Minor}.{Build}.*" will update just the revision.
+
+[assembly: AssemblyVersion("0.0.0.0")]
+[assembly: AssemblyFileVersion("2.0.0.0")]
+[assembly: AssemblyInformationalVersion("2.0.0-SNAPSHOT")]
+
+// Use WebKit instead of UIWebView when rendering web view.
+[assembly: ExportRenderer(typeof(WKWebView), typeof(Xamarin.Forms.Platform.iOS.WebViewRenderer))]
```

Comment:
Are we asking customers to do this now? This was better before the change, should not release that.

Reply:
Also: https://github.com/dougnutz/app-acquaint/blob/fix/bundle-identifier/App/Acquaint.XForms/Acquaint.XForms.iOS/Acquaint.XForms.iOS.csproj#L39

Reply:
Nope this is a workaround and I reverted this one as we discussed offline.